src/components/Navbar/Navbar.js

```diff
@@ -102,19 +108,15 @@ export function Navbar() {
         <div className={theme(desktopMenu, desktopMenuDark)}>
           <div className={theme(desktopMenuItems, desktopMenuItemsDark)}>
             <div className={desktopMenuItem}>
-              <Link
-                activeClassName={activeLink}
-                className={menuLinks}
-                to={SUB_PAGES.PROJECTS}
-              >
+              <a className={menuLinks} onClick={() => scrollToProjects()}>
```

Comment:
jeśli nie podajemy argumentów do funkcji to możemy ja wywołać po prostu: `onClick={scrollToProjects}`
oraz jeśli to jest scroll a nie link to powinno to być w `button` a nie w `a`